Django Cheatsheets.md

```diff
@@ -0,0 +1,294 @@
+# Django Cheat Sheet
```

Comment:
CheatSheet metadata is missing, please refer to other cheat sheets to see how to add that. Also please use all lowercase letters for file name